core/src/main/kotlin/org/neo4j/graphql/Predicates.kt

```diff
@@ -205,6 +205,8 @@ enum class FieldOperator(
     C("_contains", "CONTAINS", { lhs, rhs -> lhs.contains(rhs) }),
     SW("_starts_with", "STARTS WITH", { lhs, rhs -> lhs.startsWith(rhs) }),
     EW("_ends_with", "ENDS WITH", { lhs, rhs -> lhs.endsWith(rhs) }),
+    MA("_matches", "=~", {lhs, rhs -> lhs.matches(rhs) }),
```

Comment:
I would suggest to call it `MATCHES`, since its only usage is for the Cypher parameters. This would make the code more readable.

Reply:
Done. Thanks, @Andy2003